main.js

```diff
@@ -43,7 +43,7 @@ function login() {
 }
 
 function app(user) {
-  alert("Hello, " + user.displayName);
+  //alert("Hello, " + user.displayName);
```

Comment:
This function does nothing now. You should remove it instead.

Reply:
Oh yes! Sorry! I forgot I commented that out! I can definitely remove it though!